apheleia.el

```diff
@@ -6,7 +6,7 @@
 ;; Created: 7 Jul 2019
 ;; Homepage: https://github.com/raxod502/apheleia
 ;; Keywords: tools
-;; Package-Requires: ((emacs "26"))
+;; Package-Requires: ((emacs "27.1"))
```

Comment:
package-lint was yelling at me because of `exec-path`

Reply:
Interesting, that sounds like a bug we should report upstream to package-lint, because `exec-path` is definitely not new in 27.1 (see `docker run -it --rm silex/emacs:26` to verify). The CI for Apheleia doesn't use package-lint, though, so that should be a nonblocking issue.